package.json

```diff
@@ -154,7 +154,7 @@
     "cssnano": "^5.0.8",
     "cy-mobile-commands": "^0.3.0",
     "cy-tr-reporter": "^1.2.5",
-    "cypress": "^9.5.0",
+    "cypress": "^9.5.1",
```

Comment:
We should let the front end testing tools team do these kind of updates

Reply:
Interesting.. I didn't manually make this change, so some wires could have been crossed when I was rebasing/fixing some tests. I'll check it out; thank you for spotting this!

Reply:
adding @ddzz